Sources/NetworkProtection/ExtensionMessage/ExtensionMessage.swift

```diff
@@ -40,6 +45,11 @@ public enum ExtensionMessage: RawRepresentable {
         case simulateConnectionInterruption
     }
 
+    // This is actually an improved way to send messages.
+    // Please avoid adding new messages to this enum, and instead
+    // add them to `ExtensionRequest`
+    case request(_ request: ExtensionRequest)
```

Comment:
Comment has it... `ExtensionRequest` is meant to replace `ExtensionMessage`.  I'll keep moving towards this.